inference-engine/src/preprocessing/arm_neon/ie_preprocess_gapi_kernels_neon.hpp

```diff
@@ -0,0 +1,209 @@
+// Copyright (C) 2018-2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ie_preprocess_gapi_kernels.hpp"
+#include "ie_preprocess_gapi_kernels_impl.hpp"
+#include  <type_traits>
+
+namespace InferenceEngine {
+namespace gapi {
+namespace kernels {
+namespace neon {
+
+using C3 = std::integral_constant<int, 3>;
+using C4 = std::integral_constant<int, 4>;
+//-----------------------------------------------------------------------------
+
+typedef MapperUnit<float,   int> MapperUnit32F;
+typedef MapperUnit<Q0_16, short> MapperUnit8U;
+
+void calcRowArea_8U(uchar dst[], const uchar *src[], const Size &inSz, const Size &outSz,
+                    Q0_16 yalpha, const MapperUnit8U& ymap, int xmaxdf, const short xindex[],
+                    const Q0_16 xalpha[], Q8_8 vbuf[]);
+
+void calcRowArea_32F(float dst[], const float *src[], const Size &inSz, const Size &outSz,
+                     float yalpha, const MapperUnit32F& ymap, int xmaxdf, const int xindex[],
+                     const float xalpha[], float vbuf[]);
+
+#if USE_CVKL
+void calcRowArea_CVKL_U8_SSE42(const uchar  * src[],
+                                     uchar    dst[],
+                               const Size   & inSz,
+                               const Size   & outSz,
+                                     int      y,
+                               const uint16_t xsi[],
+                               const uint16_t ysi[],
+                               const uint16_t xalpha[],
+                               const uint16_t yalpha[],
+                                     int      x_max_count,
+                                     int      y_max_count,
+                                     uint16_t vert_sum[]);
+#endif
+
+//-----------------------------------------------------------------------------
+
+// Resize (bi-linear, 8U)
+void calcRowLinear_8U(uint8_t *dst[],
+                      const uint8_t *src0[],
+                      const uint8_t *src1[],
+                      const short    alpha[],
+                      const short    clone[],
+                      const short    mapsx[],
+                      const short    beta[],
+                      uint8_t  tmp[],
```

Comment:
Please, fix indention here and in many similar cases below. I mean when you call `foo()`, please align all the input arguments in a single manner.

Reply:
done.